utils/utils.go

```diff
@@ -25,3 +25,22 @@ func ScaleFixed(number *big.Int) float64 {
 	retF, _ := new(big.Float).Quo(new(big.Float).SetInt(number), big.NewFloat(1e18)).Float64()
 	return retF
 }
+
+func BoolToMetric(value bool) float64 {
+	if value {
+		return 1
+	}
+	return 0
+}
+
+func Mean(xs []*big.Int) float64 {
```

Comment:
I'd probably pick a more accurate name for this since it is not really just calculating the mean, but the mean of an array of fixidity values